boards/apollo3/lora_things_plus/Makefile

```diff
@@ -55,3 +55,10 @@ test-atecc508a:
 	$(Q)cp layout.ld $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/release/deps/
 	$(Q)cp ../../kernel_layout.ld $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/
 	$(Q)OBJCOPY=${OBJCOPY} PORT=$(PORT) $(CARGO) test $(NO_RUN) --bin $(PLATFORM) --release --features atecc508a
+
+.PHONY: test-chirp_i2c_moisture
+test-chirp_i2c_moisture:
+	mkdir -p $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/release/deps/
+	$(Q)cp layout.ld $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/release/deps/
+	$(Q)cp ../../kernel_layout.ld $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/
```

Comment:
I'm trying to figure this out for the other PR, but why do we need to copy around linker files to run tests?

Reply:
It seems like we don't need it any more, I have removed them